examples/spec/cassandra.ini

```diff
@@ -0,0 +1,1978 @@
+
+[/gc_log_threshold_in_ms]
+;type = unsigned_long
+check/type = unsigned_long
+default = 200 
+description = "
+# By default, Cassandra logs GC Pauses greater than 200 ms at INFO level
+# This threshold can be adjusted to minimize logging if necessary"
+
+[/enable_user_defined_functions]
+;type = boolean
+check/type = boolean
+default = false
+description = "
+# If unset, all GC Pauses greater than gc_log_threshold_in_ms will log at
+# INFO level
+# UDFs (user defined functions) are disabled by default.
+# As of Cassandra 3.0 there is a sandbox in place that should prevent execution of evil code."
+
+[/enable_scripted_user_defined_functions]
+;type = boolean
+check/type = boolean
+default = false
+description = "
+# Enables scripted UDFs (JavaScript UDFs).
+# Java UDFs are always enabled, if enable_user_defined_functions is true.
+# Enable this option to be able to use UDFs with 'language javascript' or any custom JSR-223 provider.
+# This option has no effect, if enable_user_defined_functions is false."
+
+[/windows_timer_interval]
+;type = float
+check/type = float
+default = 1
+description = "
+# The default Windows kernel timer and scheduling resolution is 15.6ms for power conservation.
+# Lowering this value on Windows can provide much tighter latency and better throughput, however
+# some virtualized environments may see a negative performance impact from changing this setting
+# below their system default. The sysinternals 'clockres' tool can confirm your system's default
+# setting."
+
+[/transparent_data_encryption_options/_]
+description = "# Enables encrypting data at-rest (on disk). Different key providers can be plugged in, but the default reads from
+# a JCE-style keystore. A single keystore can hold multiple keys, but the one referenced by
+# the 'key_alias' is the only key that will be used for encrypt opertaions; previously used keys
+# can still (and should!) be in the keystore and will be used on decrypt operations
+# (to handle the case of key rotation).
+#
+# It is strongly recommended to download and install Java Cryptography Extension (JCE)
+# Unlimited Strength Jurisdiction Policy Files for your version of the JDK.
+# (current link: http://www.oracle.com/technetwork/java/javase/downloads/jce8-download-2133166.html)
+#
+# Currently, only the following file types are supported for transparent data encryption, although
+# more are coming in future cassandra releases: commitlog, hints"
+
+[/transparent_data_encryption_options/enabled]
+;type = boolean
+check/type = boolean
+default = false
+
+[/transparent_data_encryption_options/chunk_length_kb]
+;type = unsigned_long
+check/type = unsigned_long
+default = 64 
+
+[/transparent_data_encryption_options/cipher]
+;type = string
+check/type = string
+default = AES/CBC/PKCS5Padding
+
+[/transparent_data_encryption_options/key_alias]
+;type = string
+check/type = string
+default = testing:1
+
+[/transparent_data_encryption_options/key_provider]
+
+[/transparent_data_e
```

Comment:
Seems to be incomplete?

Reply:
actually not:
The description of the yaml contained lines affecting multiple subsettings
```
# .... some long description affecting all subsettings and explaining everything
transparent_data_encryption_options:
    enabled: false
    chunk_length_kb: 64
    cipher: AES/CBC/PKCS5Padding
    key_alias: testing:1
    # CBC IV length for AES needs to be 16 bytes (which is also the default size)
    # iv_length: 16
    key_provider: 
      - class_name: org.apache.cassandra.security.JKSKeyProvider
        parameters: 
          - keystore: conf/.keystore
            keystore_password: cassandra
            store_type: JCEKS
            key_password: cassandra
```
With this method I have the description for the subsettings too without duplication
Unfortunately you only see the description until you set a value (#2448)